deploy/services-demo/conf/cargohold.demo-docker.yaml

```diff
@@ -11,3 +11,6 @@ aws:
 settings:
   maxTotalBytes: 27262976
   downloadLinkTTL: 300    # Seconds
+
+logLevel: Info
```

Comment:
I believe these will also have to be added here 😄 

Thanks for doing this; makes it a lot easier to configure on k8s.

Reply:
Yeah, see https://github.com/wireapp/wire-server-deploy/pull/32